services/networking/vpc/controls.yaml

```diff
@@ -1,20 +1,20 @@
 common_controls:
-  - CCC.C01 # Prevent unencrypted requests
-  - CCC.C03 # Implement multi-factor authentication (MFA) for access
-  - CCC.C04 # Log all access and changes
-  - CCC.C05 # Prevent access from untrusted entities
-  - CCC.C06 # Prevent deployment in restricted regions
+  - CCC.C01  # Prevent unencrypted requests
+  - CCC.C03  # Implement multi-factor authentication (MFA) for access
+  - CCC.C04  # Log all access and changes
+  - CCC.C05  # Prevent access from untrusted entities
+  - CCC.C06  # Prevent deployment in restricted regions
 
 controls:
   - id: CCC.VPC.C01
-    title: Skip Default Network Creation
+    title: Restrict Default Network Creation
     objective: |
-      Prevent the automatic creation of default virtual networks and related resources during cloud
-      project initialization to avoid insecure default configurations and enforce custom network policies.
+      Restrict the automatic creation of default virtual networks and related
+      resources during subscription initialization to avoid insecure default
+      configurations and enforce custom network policies.
     control_family: Network Security
     threats:
-      - CCC.VPC.TH01
-      - CCC.TH01 # Access control is misconfigured (common threat)
+      - CCC.VPC.TH01  # Unauthorized Access via Insecure Default Networks
```

Comment:
The CCM control TVM-02 is specific to malware. A better mapping might be IVS-01 (Network Security).